src/services/assetService.test.ts

```diff
@@ -1,46 +1,123 @@
 import { AssetService } from "./assetService";
-import { AssetType } from "../models/applicationState";
+import { AssetType, IAssetMetadata } from "../models/applicationState";
+import MockFactory from "../common/mockFactory";
+import { AssetProviderFactory, IAssetProvider } from "../providers/storage/assetProvider";
+import { StorageProviderFactory, IStorageProvider } from "../providers/storage/storageProvider";
 
 describe("Asset Service", () => {
-    it("creates an asset from a file path", () => {
-        const path = "C:\\dir1\\dir2\\asset1.jpg";
-        const asset = AssetService.createAssetFromFilePath(path);
-
-        expect(asset).not.toBeNull();
-        expect(asset.id).toEqual(expect.any(String));
-        expect(asset.name).toEqual("asset1.jpg");
-        expect(asset.type).toEqual(AssetType.Image);
-        expect(asset.path).toEqual(path);
-        expect(asset.format).toEqual("jpg");
-    });
+    describe("Static Methods", () => {
+        it("creates an asset from a file path", () => {
+            const path = "C:\\dir1\\dir2\\asset1.jpg";
```

Comment:
Did we want to have some of this in the `MockFactory`? I realize we don't need to use it for every piece of fake data, but it might be nice to keep it consistent so you don't have so many string literals throughout the test. Maybe something like `MockFactory.createFilePath(ext)` or something like that

Reply:
Let's do a full pass on this type of stuff and create a standalone PR.